### DISCUSSION TITLE
Problem on vercel while using nextauth

### DISCUSSION THREAD

**Discussion by Kosolov325** [vercel/next.js#49411] [Help]
### Summary

Since last week i'm trying to deploy a nextjs application.. but i'm facing some troubles while in vercel enviroment. For example, locally using vercel dev i'm being able to authenticate trought my application using "authenticate" in "[...nextauth].ts", but when deploying in vercel i'm having these messages:
![image](https://user-images.githubusercontent.com/58604882/236691002-17f90e44-3261-400c-86d4-8f5d4fa9ec06.png)
![image](https://user-images.githubusercontent.com/58604882/236692365-3d20ab9e-1aa9-43e9-a283-11610243d23e.png)

I personally think that i'm missing something, but no clue. I've read the docs and it says that you need at least NEXTAUTH_SECRET to use nextauth, i also tried NEXTAUTH_URL.. but still having troubles, i would like to test it in preview at least tho.

I seek for answers here and none of them managed to help me then i opened this thread.

![image](https://user-images.githubusercontent.com/58604882/236692377-07ce9190-98f3-4d59-b4dd-659e518dc3f6.png)

![image](https://user-images.githubusercontent.com/58604882/236691107-34938cdd-05e3-4b4b-b4ae-1cb29d804ad4.png)

![image](https://user-images.githubusercontent.com/58604882/236690954-f3d81ab1-edb4-4a8c-9fe6-77160811ee31.png)

**Comment by itsCynic** [comment]
Having the same issue

**Comment by Kosolov325** [comment]
No idea why, but axios seems to be crashing when being in used in this occasion, just switched to fetch.
